cmplibrary/src/androidTest/java/com/sourcepoint/cmplibrary/campaign/CampaignManagerImplTest.kt

```diff
@@ -178,4 +181,20 @@ class CampaignManagerImplTest {
 
         cm.metaDataResp.assertNull()
     }
+
+    @Test
+    fun `GIVEN_an_expired_GDPR_and_a_valid_CCPA_campaign_DELETE_only_the_GDPR_consent`() {
+        val json = JSONObject("v7/expired_gdpr_valid_ccpa.json".file2String())
+        appContext.storeTestDataObj(json.toList())
+        ds.deleteGdprConsent()
+        appContext.spEntries().toList().find { it.first.contains("GDPR") }.assertNull()
```

Comment:
I think we should assert that `CCPA` campaign is `not null` in this case, since we want to make sure that only `GDPR` campaign was deleted

Reply:
the test is: GIVEN_an_expired_GDPR_and_a_valid_CCPA_campaign_DELETE_only_the_GDPR_consent
this mean that there shouldn’t be keys belonging to the GDPR campaign